helpers/dark-mode.js

```diff
@@ -0,0 +1,85 @@
+function darkFilesFilter(token) {
```

Comment:
now I see why... maybe we should create 2 different files for modes and keep the dark mode here

Reply:
nice!
I refactored making the file more generic, see if it works pls